src/runtime/pkg/katautils/create.go

```diff
@@ -163,9 +163,9 @@ func CreateSandbox(ctx context.Context, vci vc.VC, ociSpec specs.Spec, runtimeCo
 	ociSpec.Annotations["nerdctl/network-namespace"] = sandboxConfig.NetworkConfig.NetworkID
 	sandboxConfig.Annotations["nerdctl/network-namespace"] = ociSpec.Annotations["nerdctl/network-namespace"]
 
-	// The value of this annotation is sent to the sandbox using SetPolicy.
-	delete(ociSpec.Annotations, vcAnnotations.Policy)
-	delete(sandboxConfig.Annotations, vcAnnotations.Policy)
+	// The value of this annotation is sent to the sandbox using init data.
+	delete(ociSpec.Annotations, vcAnnotations.Initdata)
```

Comment:
@Xynnn007 , is it a problem if the Host deletes the Init Data annotation?

The Host has been deleting the similar Policy annotation because:
- That data is sent to the Guest in a different format.
- The annotation/Policy is relatively large.

Reply:
My reasoning was this was ok to do here because this is after we decode the annotation in https://github.com/kata-containers/kata-containers/blob/main/src/runtime/pkg/oci/utils.go#L940